samples/fabric/go-cli/.env

```diff
@@ -0,0 +1,6 @@
+DEFAULT_CHANNEL=mychannel
+DEFAULT_CHAINCODE=interop
+MEMBER_CREDENTIAL_FOLDER=/home/knarayan/PRs/relaysdk/weaver-dlt-interoperability/samples/fabric/fabric-cli/src/data/credentials
+LOCAL=true
+DEFAULT_APPLICATION_CHAINCODE=simplestate
+CONFIG_PATH=./config.json
```

Comment:
Should delete the `.env`, and add it to gitignore.

Reply:
Done!